elpy/impmagic.py

```diff
@@ -68,8 +68,9 @@ def sort_key(item):
         return ["from %s import %s" % (mod, var) if var else "import %s" % mod
                 for (_, mod, var) in scores]
 
-    def add_import(self, source, statement):
-        imports = importmagic.importer.Imports(self.symbol_index, source)
+    def add_import(self, source, statement, project_root):
```

Comment:
Shouldn't `project_root=None` be in the function definition instead of just `project_root`? That way in line 73 you can just pass `project_root` without setting it to `None`.